internal/sourcemap/elasticsearch.go

```diff
@@ -59,7 +59,12 @@ type esSourcemapResponse struct {
 		Hits []struct {
 			Source struct {
 				Sourcemap struct {
```

Comment:
> Align with the UI team on the document structure and whether the current sourcemap index and structure can be reused, and only the field for the content hash would be added.

**issue**: I think we should reach out now that we have laid down the impl on our side.

Reply:
See https://github.com/elastic/kibana/pull/147208